### PR TITLE
chore: remove binaries which don't need to be released (e.g. stripped) and don't need to to uploaded to the CDN

### PR DESCRIPTION
**Pull request by rumenov**

### CODE REVIEW

File: publish/binaries/BUILD.bazel [hunk 1]
```diff
@@ -46,15 +45,11 @@ BINARIES = {
     "ic-consensus-pool-util": "//rs/artifact_pool:ic-consensus-pool-util",
     "ic-crypto-csp": "//rs/crypto:ic-crypto-csp",
     "ic-nns-init": "//rs/nns/init:ic-nns-init",
-    "ic-prep": "//rs/prep:ic-prep",
     "ic-recovery": "//rs/recovery:ic-recovery",
     "ic-regedit": "//rs/registry/regedit:ic-regedit",
     "ic-registry-replicator": "//rs/orchestrator/registry_replicator:ic-registry-replicator",
     "ic-replay": "//rs/replay:ic-replay",
-    "ic-rosetta-api": "//rs/rosetta-api:ic-rosetta-api",
```
Comment (rumenov) [inline publish/binaries/BUILD.bazel:54]: @mbjorkqvist @NikolasHai , I believe you release the rosetta nodes using different build targets than this one but please confirm

File: publish/binaries/BUILD.bazel [hunk 2]
```diff
@@ -46,15 +45,11 @@ BINARIES = {
     "ic-consensus-pool-util": "//rs/artifact_pool:ic-consensus-pool-util",
     "ic-crypto-csp": "//rs/crypto:ic-crypto-csp",
     "ic-nns-init": "//rs/nns/init:ic-nns-init",
-    "ic-prep": "//rs/prep:ic-prep",
```
Comment (basvandijk) [inline publish/binaries/BUILD.bazel:49]: I think UTOPIA downloads ic-prep from the CDN so it has to stay. @michael-weigelt can you confirm?

Comment (rumenov) [reply publish/binaries/BUILD.bazel:49]: Is there a reason to be on CDN ? Can't we just use `bazel run` from head ?

Comment (michael-weigelt) [reply publish/binaries/BUILD.bazel:49]: Yes, we need it on the CDN. The reason is that the utopia projects are in different repos in a different org, so we cannot build it ourselves.

File: publish/binaries/BUILD.bazel (hunk 1 above)
Comment (mbjorkqvist) [reply publish/binaries/BUILD.bazel:54]: I also believe that this is the case, but @NikolasHai would be the best person to confirm.

**Review by basvandijk — APPROVED** [review]

File: publish/binaries/BUILD.bazel (hunk 1 above)
Comment (NikolasHai) [reply publish/binaries/BUILD.bazel:54]: We release ICP rosetta through a docker container, like [this](https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/rosetta-api/BUILD.bazel?L216). We never make a release for the binary directly.

Comment (rumenov) [reply publish/binaries/BUILD.bazel:54]: ok, so I will proceed and merge the PR

**Pull request merged**